app/server/public/timeshare.ejs

```diff
@@ -439,6 +439,9 @@
         // console.log("this is paused")
       };
       const addmin = (min, status) => {
+        //sets minute range for min, checks if number. toFixed(4) rounds min to 4 decimal places. 
+        if (min < .1 || min > 99 || isNaN(min) ) return
+        min = min.toFixed(4)
```

Comment:
can you add validation for 'break' and 'work', so they don't send 'elephant'

Reply:
yes